lib/modules/manager/composer/__fixtures__/composer6.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "acme/git-sources",
+  "description": "Fetch Packages via bitbucket-tags",
+  "repositories": [
+    {
+      "name": "awesome/bitbucket-repo1",
+      "type": "vcs",
+      "url": "https://bitbucket.org/awesome/bitbucket-repo1.git"
+    },
+    {
+      "name": "awesome/bitbucket-repo2",
+      "type": "vcs",
+      "url": "git@bitbucket.org/awesome/bitbucket-repo2.git"
+    },
+    {
+      "name": "awesome/bitbucket-repo3",
+      "type": "vcs",
+      "url": "git@bitbucket.org/awesome/bitbucket-repo3"
+    }
+  ],
```

Comment:
Only for clarification: I have added three types of URL formats, that are covered by the regex-based recognition.